### PR TITLE
Update multi-chain.mdx

### PR DESCRIPTION
**Pull request by Saviour1001**

### CODE REVIEW

**Comment by Saviour1001** [comment]
merge along with [this](https://github.com/Web3Auth/examples/pull/46)

**Review by shahbaz17 — CHANGES_REQUESTED** [review]
Please update the method name to these tabs too:
From: `{ method: "eth_privateKey" }` to `{ method: "eth_private_key" }`

Screenshot 2023-01-09 at 6 44 00 PM

**Comment by Saviour1001** [comment]
Done !

File: docs/connect-blockchain/multi-chain.mdx [hunk 1]
```diff
@@ -399,8 +399,28 @@ import algosdk from "algosdk";
...
+// Polkadot
+import { Keyring } from "@polkadot/api";
+import { cryptoWaitReady } from "@polkadot/util-crypto";
+
+/*
+  Use code from the above Initializing Provider here
+*/
+
+const keyring = new Keyring({ ss58Format: 42, type: "sr25519" });
+const privateKey = provider.request({ method: "private_key" });
```
Comment (yashovardhan) [inline docs/connect-blockchain/multi-chain.mdx:422]: method will be `eth_private_key` here as well, you're using the `eip155` provider not the `other` provider

**Review by yashovardhan — CHANGES_REQUESTED** [review]
Ideally on this page we should change our strategy from using the eip155 provider to other provider, this way everything will be consistent.

In the current way we're showcasing how to convert eth dominant system to other blockchains.

@shahbaz17 what do you think?

File: docs/connect-blockchain/multi-chain.mdx (hunk 1 above)
Comment (shahbaz17) [reply docs/connect-blockchain/multi-chain.mdx:422]: Yes, I agree. We should be consistent here. Though, this works but should be from the eip155 provider to other providers. 

@Saviour1001, while you are at it, please update the page's wording to match that too.

Comment (Saviour1001) [reply docs/connect-blockchain/multi-chain.mdx]: updated it

Comment (shahbaz17) [reply docs/connect-blockchain/multi-chain.mdx]: Have we updated the example app to reflect the same?

Comment (Saviour1001) [reply docs/connect-blockchain/multi-chain.mdx]: yep @shahbaz17 , its `eth_private_key` for the example as well

Comment (shahbaz17) [reply docs/connect-blockchain/multi-chain.mdx]: Cool. Thanks for clarifying.

**Review by shahbaz17 — APPROVED** [review]
LGTM

**Comment by shahbaz17** [comment]
Over to @yashovardhan

**Review by yashovardhan — APPROVED** [review]
For now, merging it. However we should update our example and documentation to use other chainnamespace rather than eth. It just shows that we're eth dominant product.

**Pull request merged**